marklogic-data-hub-central/ui/src/pages/Browse.tsx

```diff
@@ -28,7 +27,27 @@ import {CSSProperties} from "react";
 import GraphViewExplore from "../components/explore/graph-view-explore";
 import {HCTooltip, HCSider} from "@components/common";
 
+const CARD = "Card";
+const GRAPH = "Graph";
 
+const FIRST_TIME_VIEW = [
```

Comment:
Let's move these constant config variables(line 30 to 38) to a helper config file under src/config under explorer.config.js (create this file if it doesn't exist).